### PR TITLE
Remove file system based data repo

### PR DESCRIPTION
**Pull request by jeromekelleher**
This removes the FileSystemRepository and FileSystemDataset, and moves fully over to the SQL repo based code for tests. The commit adds in a new file ``repo.db`` into ``tests/data`` which exactly mimics the structure of the old dataset. I've left the original hierarchy for now to minimise the diffs, but we should flatten the structure of the test data at some point.

After this is commited to the sql repo branch I think we should look to merge into master ASAP.

Depends on #1126. Issue #1112.

### CODE REVIEW

File: ga4gh/frontend.py [hunk 1]
```diff
@@ -223,14 +223,9 @@ def configure(configFile=None, baseConfig="ProductionConfig",
...
-        # Temporary hack. While we still have the FileSystemDataRepository
-        # for testing, only use the SQL repo if the path is a directory.
         path = os.path.join(dataSource.netloc, dataSource.path)
-        if os.path.isdir(path):
-            dataRepository = datarepo.FileSystemDataRepository(path)
-        else:
-            dataRepository = datarepo.SqlDataRepository(path)
-            dataRepository.open("r")
+        dataRepository = datarepo.SqlDataRepository(path)
+        dataRepository.open("r")
```
Comment (dcolligan) [inline ga4gh/frontend.py]: READ_MODE

File: tests/datadriven/test_sequenceAnnotations.py [hunk 2]
```diff
@@ -110,8 +112,10 @@ def __init__(self, featureSetLocalName, dataPath):
         :return:
         """
         self._dataset = datasets.Dataset(_datasetName)
-        self._repo = datarepo.FileSystemDataRepository("tests/data")
-        self._sequenceOntology = self._repo.getOntology("sequence_ontology")
+        self._repo = datarepo.SqlDataRepository(paths.testDataRepo)
+        self._repo.open("r")
```
Comment (dcolligan) [inline tests/datadriven/test_sequenceAnnotations.py]: READ_MODE

**Comment by dcolligan** [comment]
great to move to a pure db based backend... will review again once the dependencies are merged and this is rebased

File: ga4gh/frontend.py (hunk 1 above)
Comment (jeromekelleher) [inline ga4gh/frontend.py]: Alrighty...

File: tests/datadriven/test_sequenceAnnotations.py (hunk 2 above)
Comment (jeromekelleher) [inline tests/datadriven/test_sequenceAnnotations.py]: Done.

**Comment by jeromekelleher** [comment]
Closing this in favour of #1155 --- this branch got a bit mangled.

**Pull request closed without merging**